spark/src/main/scala/org/apache/spark/sql/delta/util/JsonUtils.scala

```diff
@@ -28,6 +29,13 @@ object JsonUtils {
     _mapper.setSerializationInclusion(Include.NON_ABSENT)
     _mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
     _mapper.registerModule(DefaultScalaModule)
+
```

Comment:
Maybe add a quick comment about us overriding the default. (e.g. why int.max and not long.maxvalue). Was this the old default value?

Reply:
I added a comment. Since the parameter is of type int, I think commenting why it is not long.maxvalue is very redundant